time/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "time"
+version = "0.1.0"
+authors = ["Vaibhav Yenamandra <v@calloc.net>"]
+build = "build.rs"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "^2.33.0", features = ["yaml", "wrap_help"] }
```

Comment:
Looks like you're using yaml feature of clap. We dropped that due some problems with it. You can find the template used now [here](https://github.com/GrayJack/coreutils/tree/dev/.template)